### DISCUSSION TITLE
Rust in serverless  framework

### DISCUSSION THREAD

**Discussion by LuWroblewski** [serverless/serverless#12625] [Q&A] [answered]
have date to serverless framework compatibility with rust ?

**Comment by pgrzesik** [comment, marked as answer]
@LuWroblewski Serverless Framework is generally compatible with Rust, I remember using it in the past for experimentation with https://www.serverless.com/plugins/serverless-rust plugin. Are you looking for any specific Rust support within Framework?

**Reply by LuWroblewski** [reply]
oh nice, 

When I went to use it a while ago they told me it didn't have it

thx <3

**Reply by LuWroblewski** [reply]
i only use for create API using axum and tokio etc and create schedule functions

**Reply by pgrzesik** [reply]
I think there shouldn't be any limitations when it comes to your Rust code functionalities, I didn't test the Rust plugin with v4 release, but I think it should work just fine

**Reply by LuWroblewski** [reply]
i test and give a comments for us
